src/main/java/io/kestra/plugin/notifications/twilio/TwilioAlert.java

```diff
@@ -0,0 +1,109 @@
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.Task;
+import io.kestra.core.models.tasks.VoidOutput;
+import io.kestra.core.runners.RunContext;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.client.netty.DefaultHttpClient;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotBlank;
+import java.net.URI;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Send a Twilio message using an notification API",
+    description = "Add this task to a list of `errors` tasks to implement custom flow-level failure notifications. Check the <a href=\"https://www.twilio.com/docs/notify/send-notifications#sending-notifications\">Twilio documentation</a> for more details.."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Send a Twilio notification on a failed flow execution",
+            full = true,
+            code = """
+                id: unreliable_flow
+                namespace: prod
+
+                tasks:
+                  - id: fail
+                    type: io.kestra.plugin.scripts.shell.Commands
+                    runner: PROCESS
+                    commands:
+                      - exit 1
+
+                errors:
+                  - id: alert_on_failure
+                    type: io.kestra.plugin.notifications.twilio.TwilioAlert
+                    url: "{{ secret('TWILIO_ALERT') }}" # https://notify.twilio.com/v1/Services/ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Notifications
+                    payload: |
+                      {
+                        "identity": "0000001"
+                      }
+                """
+        ),
+        @Example(
+            title = "Send a Twilio message via incoming notification API",
+            full = true,
+            code = """
+                id: twilio_alert
+                namespace: dev
+
+                tasks:
+                  - id: send_twilio_message
+                    type: io.kestra.plugin.notifications.twilio.TwilioAlert
+                    url: "{{ secret('TWILIO_ALERT') }}"
+                    payload: |
+                      {
+                        "identity": "0000001"
+                      }
+                """
+        ),
+    }
+)
+public class TwilioAlert extends Task implements RunnableTask<VoidOutput> {
+
+    @Schema(
+        title = "Twilio notification URL"
+    )
+    @PluginProperty(dynamic = true)
+    @NotBlank
+    protected String url;
+
+    @Schema(
+        title = "Twilio message payload"
+    )
+    @PluginProperty(dynamic = true)
+    protected String payload;
+
+    protected String accountSID;
+
+    protected String authToken;
```

Comment:
Missing `@PluginProperty` and `@Schema` and possibly constraint annotation as they seems mandatory